src/java.desktop/share/classes/javax/swing/plaf/synth/SynthTableUI.java

```diff
@@ -594,17 +602,18 @@ private void paintCells(SynthContext context, Graphics g, int rMin,
         } else {
             for(int row = rMin; row <= rMax; row++) {
```

Comment:
```suggestion
            for (int row = rMin; row <= rMax; row++) {
```

Reply:
it's just a spacing thing, but should these be left alone since they weren't changed with the fix?

Reply:
Yeah, not sure whether update this or not. Anyhow will update for uniformity.